docs/_data/table/common-configuration.csv

```diff
@@ -146,6 +147,9 @@ alluxio.zookeeper.leader.path,"/alluxio/leader"
 alluxio.zookeeper.session.timeout,"60s"
 aws.accessKeyId,""
 aws.secretKey,""
+awt.toolkit,""
```

Comment:
these 3 look a bit strange to me, do you know where they come from?
i found a `File.separator` call in TarUtils.java, but that was added a year ago
i also could not find any matching strings for `file.encoding`

Reply:
that could be some environment variables that not used by alluxio. For the second commit, i delete some of the parameters that maybe from jvm or the underlying package. I will remove these properties in future PR